pkg/log/default.go

```diff
@@ -0,0 +1,41 @@
+package log
+
+import (
+	"go.uber.org/zap"
```

Comment:
Any reason to bring in a zap dependency? I would probably just have this use `log`, `fmt` or something?

Reply:
Ah I did this cause it maps nicely with the different levels and provides structured logs. I can move back to `fmt` if we want @snowp

Reply:
Ah that's fair, I'll make the change then so we don't throw in some unexpected behavior where it's not needed